elastic/kibana
# Remove nginx section from host asset details page

Closes #167739 

## Summary

This PR removes the Nginx section from the Node Details page for a host (for all hosts running Nginx)


| Before | After |
| ------ | ------ |
| ![image](https://github.com/elastic/kibana/assets/14139027/62a3190f-976b-42d6-9c21-1ebc72b68a6e) |  ![image](https://github.com/elastic/kibana/assets/14139027/fb4938b7-1a70-4776-a53c-def531407d31) |

## Testing 

- Run nginx on a host (Check the guide added [here](https://github.com/elastic/kibana/issues/162958#issuecomment-1705377975)) 
- Open the asset details page for this host
- Nginx section should not be visible
